src/js/shield_defs.js

```diff
@@ -2997,7 +3065,22 @@ export function loadShields(shieldImages) {
   // Kosovo
   shields["XK:motorway"] = wideHexagonGreenShield;
 
-  //Ref-specific cases. Additional entries should be documented in CONTRIBUTE.md
+  // OCEANIA
+
+  // New Zealand
+  shields["NZ:SH"] = {
+    backgroundImage: shieldImages.shield40_nz_sh,
```

Comment:
There are some three-digit routes too. I assume you based this image on the Interstate or Business Interstate shield, so it should be pretty straightforward to make a three-digit one too?

Reply:
I did indeed base it on the Interstate shield :smile:

A 3-digit variant has been added:

![Screenshot from 2022-06-25 21-39-32](https://user-images.githubusercontent.com/1732117/175795839-c97ed4d0-9387-45ff-bfd1-740df8c1f5af.png) ![Screenshot from 2022-06-25 21-39-54](https://user-images.githubusercontent.com/1732117/175795840-1a220230-ca73-4606-bd57-0bfe087bf152.png)